complete/src/main/java/hello/controllers/MainController.java

```diff
@@ -0,0 +1,37 @@
+package hello.controllers;
```

Comment:
I don't see any value in extra packages in this guide (and 3 packages for one entity is definitely a smell). Can we just put everything in the "hello" package?